bodhi/server/templates/update.html

```diff
@@ -884,7 +885,8 @@ <h5 class="m-t-2">
               % if build.ci_url:
                 <a href="${build.ci_url}">${self.util.ci_status2html(build.ci_status.description) | n}</a>
               % else:
-                ${self.util.ci_status2html(build.ci_status.description) | n}
+                ${self.util.ci_status2html(build.ci_status.description
+                  if build.ci_status else None) | n}
```

Comment:
Another way that could be nice to do it may be to alter ```ci_status2html()``` so that it accepts the status and knows to use the description attribute if it is not None. This is a fine way to do it too though.